Gems/AtomTressFX/CMakeLists.txt

```diff
@@ -5,3 +5,159 @@
 # SPDX-License-Identifier: Apache-2.0 OR MIT
 #
 #
+
+set(GemUUID d4a3e3d6b9b14e12982c774f35f48a7c)
```

Comment:
Setting the GemUUID doens't seem necesary, its not used?

Reply:
I believe that this is the UUID you'd get if the gem is requested but does not exist / included, or am I wrong? 
I might be wrong and if you know for a fact that this is not required I can remove this.

Reply:
I don't think its necessary.
If you need to discuss how to structure this more standardly please get ahold of me.

Reply:
will remove and test that all works as before, tnx.